integration/framework/scenario.go

```diff
@@ -258,7 +140,23 @@ func (s *Scenario) WaitReady(services ...string) error {
 	return nil
 }
 
-func (s *Scenario) Shutdown() {
+func (s *Scenario) Close() {
+	if s == nil {
+		return
+	}
+
+	s.shutdown()
+	s.clean()
+}
+
+// TODO(bwplotka): Add comments. Return err?
```

Comment:
Previously, the `Shutdown()` didn't intentionally return err (but was just logging it) because even if it returns err there's no much the caller can do.

Reply:
fair